src/normalize/pathoverlay.ts

```diff
@@ -122,7 +140,8 @@ export class PathOverlayNormalizer implements NonFacetUnitNormalizer<UnitSpecWit
           ...markDef
         }),
         // drop shape from encoding as this might be used to trigger point overlay
-        encoding: omit(encoding, ['shape'])
+        // drop label from encoding when not having detail (connected scatter plot)
```

Comment:
What do you mean "when having detail (connected scatterplot)"?
Given the code below doesn't do any conditional check.